src/index.ts

```diff
@@ -1,22 +1,47 @@
+import { InteractionResponseType, InteractionType } from "discord-interactions";
 import { Router } from "itty-router";
+import * as error from "./constants/errors";
 import { Env } from "./typeDefinitions/default.types";
+import { discordMessageRequest } from "./typeDefinitions/discordMessage.types";
+import JSONResponse from "./utils/JsonResponse";
+import { verifyBot } from "./utils/verifyBot";
 
 const router = Router();
 
 router.get("/", async () => {
-  return new Response(`{message: "Welcome to our discord Bot Server 👋"}`, {
-    status: 200,
-  });
+  return new JSONResponse(
+    { message: "Welcome to our discord Bot Server 👋" },
```

Comment:
Please move to constant

Reply:
This is just one time and won't be used anywhere else, still, do you want me to move it to constants?

Reply:
yes, we should be developing a scalable code. Tomorrow if we have to do language change then it would be easier

Reply:
Worked on it could you check and resolve the conversation